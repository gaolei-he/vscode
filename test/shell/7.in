70
/bdc/a/cbd
/ac/cd
/da/bdb/a
/bc/cca
/b/a
/bac/cdd/bab
/a/a/bca
/cdd/d/d
/ab/bba/aca
/bba/dbb
/d/bcc/adc
/a
/a
/da
/bc/db/aad
/dd/da/cc
/b/bab
/db/ddd/ac
/c/c
/ac
/bd/d/da
/dbd/cc/dd
/d/aba/b
/bab
/cdd/dc/d
/d/aac/a
/cb/d/bcc
/dcc/dad/dc
/ab/cd
/d/cad/db
/cda
/ccb/cda/b
/c/bbc
/d/cab
/bb
/cba
/adc/d
/d/db
/add
/cab/dc/a
/ca
/dd/da
/dc
/c
/da
/cc/ba
/aba/cda/c
/bca
/acb/ccc
/dbd/bab/cb
/ac/acc
/bb/b/ac
/b
/d/bdc
/db/c/d
/c/d/ac
/bcb/bb/d
/d/b/aba
/bbb
/a/b
/dbd/dd/aa
/aaa/a/bcb
/cab/b/cba
/ccd/dd
/d/bd/cbb
/bb/bc/bcd
/aa
/ab/aa/ccb
/c/c/bc
/db/aad
2
cd .
ls
