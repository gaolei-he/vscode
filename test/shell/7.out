a
aa
aaa
ab
aba
ac
acb
adc
add
b
bab
bac
bb
bba
bbb
bc
bca
bcb
bd
bdc
c
ca
cab
cb
cba
cc
ccb
ccd
cda
cdd
d
da
db
dbd
dc
dcc
dd
